src/extension.ts

```diff
@@ -47,14 +51,19 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
 
   if (crcVersion) {
     status = 'installed';
+  }
+  crcStatus = defaultStatus;
+
+  isNeedSetup = await needSetup();
+  if (crcVersion) {
     connectToCrc();
   }
 
   detectionChecks.push(...getCrcDetectionChecks(crcVersion));
 
   // create CRC provider
   const provider = extensionApi.provider.createProvider({
-    name: 'CRC',
+    name: productName,
```

Comment:
Perhaps we should also offer a way to map he preset name with a clear `Name` and `Description` in a single file.